scripts/h5peditor-av.js

```diff
@@ -683,7 +683,12 @@ H5PEditor.widgets.video = H5PEditor.widgets.audio = H5PEditor.AV = (function ($)
       name: 'Vimeo',
       regexp: /^.*(vimeo\.com\/)((channels\/[A-z]+\/)|(groups\/[A-z]+\/videos\/))?([0-9]+)/,
       aspectRatio: '16:9',
-    }
+    },
+    {
+        name: 'Echo360',
+        regexp: /^[^\/]+:\/\/(echo360[^\/]+)\/media\/([^\/]+)\/h5p.*$/i,
```

Comment:
Just a note: If Echo360 requires to have an extra link to serve videos for H5P in addition to the `public` link that one gets when intending to share the video, and that's not too confusing for Echo360 users, that regex seems fine.

Reply:
Thanks - yes Echo360 requires a separate route which triggers some minor layout changes and feature changes